graphene/src/document.rs

```diff
@@ -31,6 +31,19 @@ fn split_path(path: &[LayerId]) -> Result<(&[LayerId], LayerId), DocumentError>
 }
 
 impl Document {
+	pub fn with_content(serialized_content: String) -> Result<Self, DocumentError> {
+		let json_value: Result<serde_json::Value, serde_json::Error> = serde_json::from_str(&serialized_content);
```

Comment:
What is the reason for not just implementing/deriving Serialize for document?

Reply:
We could also think about using ron instead: https://github.com/ron-rs/ron
That would probably make our exported file look nicer

Reply:
We also need to manually implement serialize for Layer because we don't want to serialize the render cache